File: app/src/main/java/dev/ragnarok/fenrir/model/OwnerType.java
package dev.ragnarok.fenrir.model;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;


@IntDef({OwnerType.USER, OwnerType.COMMUNITY})
@Retention(RetentionPolicy.SOURCE)
public @interface OwnerType {
    int USER = 1;
    int COMMUNITY = 2;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/model/OwnerType.kt
package dev.ragnarok.fenrir.model

import androidx.annotation.IntDef

@IntDef(OwnerType.USER, OwnerType.COMMUNITY)
@Retention(AnnotationRetention.SOURCE)
annotation class OwnerType {
    companion object {
        const val USER = 1
        const val COMMUNITY = 2
    }
}